exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/internal/PrometheusMetricReaderProvider.java

```diff
@@ -33,6 +33,9 @@ public MetricReader createMetricReader(ConfigProperties config) {
       prometheusBuilder.setHost(host);
     }
 
+    prometheusBuilder.setUtf8SupportEnabled(
+        config.getBoolean("otel.exporter.prometheus.utf8", false));
```

Comment:
Is this the official property? I thought no new properties were being added to the spec?

Reply:
That doesn't answer my question... if we're adding new configuration properties, I'd like to make sure they're "official" in the spec, and we're not going off on our own, especially as there has been a moratorium set on the new configuration properties/env vars in the spec (at least, the last I heard).

Reply:
Got it :smile: 

This is the part in the Go SDK: https://github.com/open-telemetry/opentelemetry-go/blob/d0cab8666b740c975f028236610cab2663f02031/exporters/prometheus/config.go#L44-L66

@ArthurSens it seems there is no env var to set the translation strategy - is that missing?